src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,129 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+
+    private static final int SIZE_OF_LIST = 10;
+    private int size;
+    private Object[] list;
+
+    ArrayList() {
+        list = new Object[SIZE_OF_LIST];
+    }
+
     @Override
     public void add(T value) {
-
+        if (size >= list.length) {
+            increaseCapacity();
+        }
+        list[size] = value;
+        size++;
```

Comment:
```suggestion
        list[size++] = value;
```
you can write it in this way